README.md

```diff
@@ -3,40 +3,70 @@
 > Official GitHub Action for [OSSF scorecard](https://github.com/ossf/scorecard).
 
 ## Installation
-The simplest and quickest way to install Scorecards's GitHub action is from the [GitHub's marketplace](https://github.com/marketplace/actions/ossf-scorecard-action).
-**WARNING**: the action does *not* support private repositories.
+**NOTE**: The Scorecards GitHub Action does not support private repositories. Private repositories can run Scorecards manually by following the [standard installation instructions)[https://github.com/ossf/scorecard#installation].
 
-### Inputs
+To install the Scorecards GitHub Action, you need to:
 
-| Name | Required | Description |
-| ----- | -------- | ----------- |
-| `result_file` | yes | The file that contains the results. |
-| `result_format` | yes | The format in which to store the results [json \| sarif]. For GitHub's scanning dashboard, select `sarif`. |
-| `repo_token` | yes | PAT token with read-only access. Follow [these steps](#pat-token-creation) to create it. |
-| `publish_results` | recommended | This will allow you to display a badge on your repository to show off your hard work (release scheduled for Q2'22). See details [here](#publishing-results).|
+1) Authenticate by creating a PAT token and saving the token value as a repository secret (new Scorecards users only)
```

Comment:
The `new Scorecard users` part is not true. Even existing Scorecard users will have to upload their PAT as a secret. Maybe we can re-word this line as:

1) Upload a PAT as repository secret for the Scorecard workflow to use.

Reply:
How about this?  

To install the Scorecards GitHub Action, you need to:

1) Create a Personal Access Token (PAT) and save the token value as a repository secret;

(Note: current Scorecards users can reuse their existing PAT for the repository secret)

2) Set up the workflow via the GitHub UI

Reply:
LGTM, thanks!

Reply:
Updated: " (Note: If you have already installed Scorecards on your repository from the command line, you can reuse your existing PAT for the repository secret. If you not longer have access to the PAT, though, simply create a new one.)"

Hopefully clears up all confusion?